src/components/overview/CustomPieChart.js

```diff
@@ -61,27 +67,21 @@ class CustomPieChart extends React.Component {
             return;
         }
 
-        setAutoQueryPageTransition(
-            window.location.href,
-            autoQueryDataType,
-            this.props.fieldLabel,
-            data.name
-        );
+        setAutoQueryPageTransition(window.location.href, autoQueryDataType, this.props.fieldLabel, data.name);
```

Comment:
please don't alter code that isn't part of this PR - make a new PR with only the relevant changes.